### PR TITLE
fixed results of inspection "HashSet can be replaced with EnumSet"

### PR DESCRIPTION
**Pull request by dk2k**
fixed results of inspection "HashSet can be replaced with EnumSet"

### CODE REVIEW

File: components/camel-platform-http/src/main/java/org/apache/camel/component/platform/http/spi/Method.java [hunk 1]
```diff
@@ -16,11 +16,7 @@
  */
 package org.apache.camel.component.platform.http.spi;
 
-import java.util.Arrays;
-import java.util.Collections;
-import java.util.Locale;
-import java.util.Set;
-import java.util.TreeSet;
+import java.util.*;
```
Comment (davsclaus) [inline components/camel-platform-http/src/main/java/org/apache/camel/component/platform/http/spi/Method.java:19]: We do not use wildcard imports

**Review by davsclaus — CHANGES_REQUESTED** [review]
see prev comments

File: components/camel-platform-http/src/main/java/org/apache/camel/component/platform/http/spi/Method.java (hunk 1 above)
Comment (dk2k) [reply components/camel-platform-http/src/main/java/org/apache/camel/component/platform/http/spi/Method.java]: Fixed. It's IDE's optimization

Comment (dk2k) [reply components/camel-platform-http/src/main/java/org/apache/camel/component/platform/http/spi/Method.java]: Sorry for accidentally merging branch apache:main in

**Review by davsclaus — APPROVED** [review]

**Pull request merged**